create_dmd_release/build_all.d

```diff
@@ -487,6 +487,9 @@ int main(string[] args)
         extract(cacheDir~"/"~libCurl, workDir~"/windows/old-dmd/");
         // Get updated OMF import libraries
         extract(cacheDir~"/"~omflibs, workDir~"/windows/old-dmd/dmd2/windows/lib/");
+
+        // grab mingw sources
+        run("git clone --depth 1 --branch 5.0-active https://git.code.sf.net/p/mingw/mingw-org-wsl.git " ~ workDir ~ "/clones/mingw");
```

Comment:
How about checking out a specific tag or commit here to avoid breakage if the branch gets updated?

Reply:
> How about checking out a specific tag or commit here to avoid breakage if the branch gets updated?

Maybe better, but 5.0-active is a release branch with only little activity, the real work seems to happen on 5.1-trunk.

Reply:
Ah I just saw the `-active`

Reply:
Let's use a cacheable tarball (for a fixed commit), just had lots of issues with sourceforge's git. Not too convinced that their service is reliable.